upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -333,16 +332,7 @@ func NewAWSCloud(region string, tags map[string]string) (AWSCloud, error) {
 		c.ec2.Handlers.Send.PushFront(requestLogger)
 		c.addHandlers(region, &c.ec2.Handlers)
 
-		sess, err = session.NewSessionWithOptions(session.Options{
-			Config:            *config,
-			SharedConfigState: session.SharedConfigEnable,
-		})
-		if err != nil {
-			return c, err
-		}
-		c.iam = iam.New(sess, config)
-		c.iam.Handlers.Send.PushFront(requestLogger)
-		c.addHandlers(region, &c.iam.Handlers)
+		c.iam = iam.NewFromConfig(cfgV2)
```

Comment:
So we are losing 3 things:
* SharedConfigState 
* requestlogger
* addHandlers - which configures the cross-request delay

Do we think any of those are still important?

Reply:
SharedConfigState is now default: https://aws.github.io/aws-sdk-go-v2/docs/migrating/#client-construction

there is no way to disable or enable SharedConfigState anymore, only specify which shared config file to use (default `~/.aws/config`)